Sources/Sentry/SentryRequest.h

```diff
@@ -0,0 +1,50 @@
+#import "SentryDefines.h"
+#import "SentrySerializable.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+NS_SWIFT_NAME(Request)
+@interface SentryRequest : NSObject <SentrySerializable, NSCopying>
+
+// TODO: data, env
+
+/**
+ * Optional: HTTP response body size.
+ */
+@property (atomic, copy) NSNumber *_Nullable bodySize;
```

Comment:
`l`: Please change all to. We use `nullable` now everywhere. Again, we should have a linter for this, so I don't have to point it out 😁.
```suggestion
@property (nullable, atomic, copy) NSNumber * bodySize;
```

Reply:
I used other data classes as an example, a linter and a new issue to replace them all would be better, will change this one.